tests/suite_test.go

```diff
@@ -126,81 +114,81 @@ func init() {
 	flag.BoolVar(&skipCleanup, "skipCleanup", DefaultSkipCleanup, "Skip DSPA cleanup.")
 }
 
-var _ = BeforeSuite(func() {
-	ctx, cancel = context.WithCancel(context.TODO())
+func (suite *IntegrationTestSuite) SetupSuite() {
+	fmt.Println("SetupSuite started") // Debug statement
```

Comment:
// Debug statement 
Did you mean to remove this?

Reply:
Yes, removed the debug statements, just added them to test locally.